lionblog-seongil/src/components/Posts/index.jsx

```diff
@@ -14,3 +14,27 @@ export const SmallPost = ({ post }) => {
     </div>
   );
 };
+
+export const BigPost = ({ post }) => {
+  const likeBtnClick = () => {
```

Comment:
이 함수는 좋아요를 누르는 이벤트가 발생했을 때 호출되는 함수인데요 ! 이런 이벤트 핸들러 함수는 보통 handle어쩌구로 이름을 짓는 것이 일반적입니다 ~ 참고해 주세요 !
```suggestion
  const handleLike = () => {
```